gocourse03/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	Chip        RodentType = "Chip"
+	Dale        RodentType = "Dale"
+	Cheaser     RodentType = "Cheaser"
+	Hackwrench  RodentType = "Hackwrench"
+	NorthSector Sector     = "North"
```

Comment:
Між константами різних типів зазвичай додаються пусті рядки для більшої читабельності:
```suggestion
	Hackwrench  RodentType = "Hackwrench"

	NorthSector Sector     = "North"
```